src/runtime/math.py

```diff
@@ -695,7 +683,7 @@ def atanh(x: f64) -> f64:
 
 
 def expm1(x: f64) -> f64:
```

Comment:
May be, `expm1` can also become an `IntrinsicFunction`. It should just re-use `Exp` intrinsic.

Reply:
I cant think of a way to solve this in ```IntrinsicFunctionRegistry``` itself, without changing ```asr_to_llvm```.
Should we do we do something like this?
```cpp
llvm::Value* expm1(llvm::Value* x) {
    llvm::Value* exp_x = builder->CreateUnaryIntrinsic(llvm::Intrinsic::exp, x);
    llvm::Value* one = llvm::ConstantFP::get(builder->getFloatTy(), 1.0);
    llvm::Value* result = builder->CreateFSub(exp_x, one);
    return result;
}
```
If there is too much code in  ```IntrinsicFunctionRegistry```, maybe I can create a macro for ```exp()``` like you did for trigonometric functions.